client/src/app/users/user-profile.component.html

```diff
@@ -1,5 +1,5 @@
-<div fxLayout="row">
-  <div fxFlex fxFlex.gt-sm="80" fxFlexOffset.gt-sm="10">
+<div class="flex-row">
+  <div class="flex-1" fxFlex.gt-sm="80" fxFlexOffset.gt-sm="10">
```

Comment:
Looks like you've left some stuff from Angular flex-layout here. Hasn't that been removed?